### PR TITLE
Change ladder to profiles, soften wording 

### PR DESCRIPTION
**Pull request by glennawatson**
Changed to the concept of profiles rather than a ladder.

Softened language around changing between profiles and the working group process.

### CODE REVIEW

**Comment by glennawatson** [comment]
This is based around feedback from #26 and #37

File: maturity-profiles-policies.md [hunk 1]
```diff
@@ -1,26 +1,26 @@
-# .NET Foundation Maturity Ladder Policies
+# .NET Foundation Maturity Profiles Policies
```
Comment (haacked) [inline maturity-profiles-policies.md]: In the first paragraph this states:

> Any .NET open source project is welcome to participate in the maturity profiles program.

But the heading here says "Policies" I  wonder if we should change this to ".NET Foundation Maturity Profiles Program"

File: maturity-profiles-policies.md [hunk 2]
```diff
@@ -32,43 +32,43 @@ Project users cannot register on the behalf of a maintainer. They are free to re
...
 
-.NET Foundation Project Membership is separate from the ladder, but integrated in the following way:
+.NET Foundation Project Membership is separate from the maturity profiles, but integrated in the following way:
 
-* Projects do not need to formally join the Foundation to participate in the ladder.
-* Projects are given the option to join the Foundation at level 2.
-* Projects must [join the Foundation](https://github.com/dotnet/foundation/blob/master/guidance/new-projects.md) at level 3. Exemptions may be granted by the working group by specific request.
+* Projects do not need to formally join the Foundation to participate in the maturity profiles.
+* Projects are given the option to join the Foundation at [Basic security practices](maturity-profiles.md#Basic-security-practices).
+* Projects must [join the Foundation](https://github.com/dotnet/foundation/blob/master/guidance/new-projects.md) at maturity profile #3. Exemptions may be granted by the working group by specific request.
```
Comment (haacked) [inline maturity-profiles-policies.md]: Should it be "#3 and above?"

**Review by haacked — APPROVED** [review]
I like these changes.

**Review by clairernovotny — APPROVED** [review]

**Review by benaadams — APPROVED** [review]

**Comment by clairernovotny** [comment]
Just a note that this appears to miss the terms in the README.md.

**Comment by glennawatson** [comment]
I will submit another PR in a minute to address that.

**Pull request merged**